index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body> 
+   <header>Der Ozean</header>
+   
+    <main>
+    <h1>Die größten Ozeane</h1>
```

Comment:
```suggestion
    <h1>Die größten Ozeane</h1>
```
Dieses Element löschen